src/include/daos_srv/rsvc.h

```diff
@@ -130,6 +136,10 @@ struct ds_rsvc {
 	ABT_cond		s_state_cv;
 	int			s_leader_ref;	/* on leader state */
 	ABT_cond		s_leader_ref_cv;
+	bool			s_map_dist;	/* has a map dist request? */
```

Comment:
nit...ABT_cond is void * so the compiler will make this struct larger than it needs to be.  It already had this issue with s_leader_ref though.

Reply:
Right, this structure is arranged for readability, as objects of this are not so many and never put on disk or wire.